src/signing.js

```diff
@@ -7,6 +7,8 @@
 
 var actualMethods = {};
 
+export var fastSigning;
```

Comment:
I tried it before posting my first comment:
```
SyntaxError: /Users/bartek/js-stellar-base/src/signing.js: 'import' and 'export' may only appear at the top level (30:4)
```
The reason is I had an `export` statement inside the `if`statment. However, the solution seems to be setting `const` value to the value returned by a function (the first line in your example). I will change it.